Makefile

```diff
@@ -44,6 +44,15 @@ rundev: builddev
 stop:
 		@docker-compose down
 
+runtest: rundev
+		@sleep 1
```

Comment:
Is this `sleep` is really necessary to make this work?

Reply:
I found that somehow the database container was not fully ready before the next command was executed resulting in errors. This prevented that.